src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -157,13 +158,21 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
 
   const auto* thread_state_slice =
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
+  auto begin_ns = std::chrono::nanoseconds(thread_state_slice -> begin_timestamp_ns());
+  auto end_ns = std::chrono::nanoseconds(thread_state_slice -> end_timestamp_ns());
```

Comment:
There seem to be some code style problems. For example we usually don't have whitespace before and after `->`. Can you run clang-format on your change?

You should be able to do that by running `git clang-format upstream/main...HEAD`. Ping me if this doesn't work.

Reply:
I edited manually, `git clang-format upstream/main...HEAD` gave me the following error:


fatal: ambiguous argument 'upstream/main...HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

Reply:
There was another extra empty line. After removing it, the style check seems to pass.